lisa/transformers/kernel_source_installer.py

```diff
@@ -325,6 +315,23 @@ def _install_build_tools(self, node: Node, build_deps: list[str]) -> None:
                     os.install_packages(package)
             os.group_install_packages("Development Tools")
 
+            # if the kernel requires devtoolset, install its gcc
+            has_devtoolset = [
```

Comment:
variable name should be `devtoolsets`, because it saves strings of packages, not bool.